libs/atqr/domain/src/lib/services/notificationsEnums.ts

```diff
@@ -0,0 +1,40 @@
+export enum NotificationChannel {
+  Email = 'Email',
+  SMS = 'SMS',
+  Push = 'Push',
+}
+
+export enum MessageTemplateId {
+  Challenge = 'Challenge',
+  YourOfficiationHaveBeenRequested = 'YourOfficiationHaveBeenRequested',
+  YourOfficiationRequestHaveBeen = 'YourOfficiationRequestHaveBeen',
+  YouHaveBeenChallenged = 'YouHaveBeenChallenged',
+  YourContender = 'YourContender',
+  TimeToOfficiateArrived = 'TimeToOfficiateArrived',
+  YourJudgeDidNotAnswered = 'YourJudgeDidNotAnswered',
+}
+
+export enum Challenge {
+  Created = 'Created',
+  Canceled = 'Canceled',
+  Ongoing = 'Ongoing',
+  Completed = 'Completed',
+  Failed = 'Failed',
+}
+
+export enum YourOfficiationRequestHaveBeen {
+  Accepted = 'Accepted',
+  Rejected = 'Rejected',
+  Ignored = 'Ignored',
+}
+
+export enum YourContender {
+  AcceptedTheChallenge = 'AcceptedTheChallenge',
+  RejectedTheChallenge = 'RejectedTheChallenge',
+  Ignored = 'Ignored',
+  RemovedYouFromJudgeRole = 'RemovedYouFromJudgeRole',
+  CanceledTheChallenge = 'CanceledTheChallenge',
+  WereTheySuccessful = 'WereTheySuccessful',
+}
+
+
```

Comment:
Enums in Disl always start with the `enum` keyword. These were not new enums but items of `MessageTemplateId` named as I explained above

Reply:
Fix the bugs pointed out

Reply:
This conversation is no longer valid as this code has been refactored. Resolving conversation.